code/modules/antagonists/heretic/magic/mind_gate.dm

```diff
@@ -0,0 +1,52 @@
+/datum/action/cooldown/spell/pointed/mind_gate
+	name = "Mind Gate"
+	desc = "Opens your mind, and theirs."
+	background_icon_state = "bg_heretic"
+	overlay_icon_state = "bg_heretic_border"
+	button_icon = 'icons/mob/actions/actions_ecult.dmi'
+	button_icon_state = "mind_gate"
+
+	sound = 'sound/magic/curse.ogg'
+	school = SCHOOL_FORBIDDEN
+	// The spell deals brain damage so we can allow more spamming
+	cooldown_time = 5 SECONDS
+
+	invocation = "Op' 'oY 'Mi'd"
+	invocation_type = INVOCATION_WHISPER
+	spell_requirements = NONE
+	cast_range = 6
+
+	active_msg = "You prepare to open your mind..."
+
+/datum/action/cooldown/spell/pointed/mind_gate/can_cast_spell(feedback = TRUE)
+	return ..() && isliving(owner)
+
+/datum/action/cooldown/spell/pointed/mind_gate/is_valid_target(atom/cast_on)
+	return ..() && ishuman(cast_on)
+
+/datum/action/cooldown/spell/pointed/mind_gate/cast(mob/living/carbon/human/cast_on, mob/living/carbon/human/owner)
+	. = ..()
+	if(cast_on.can_block_magic(antimagic_flags))
+		to_chat(cast_on, span_notice("Your mind feels closed."))
+		to_chat(owner, span_warning("Their mind doesn't swing open, but neither does yours."))
+		return FALSE
+
+	cast_on.adjust_confusion(10 SECONDS)
+	cast_on.adjustOxyLoss(50)
```

Comment:
So it's point-click-instant KO no counterplay? seems unfair

Reply:
Lowered, but it didn't KO in testing.